airbyte-integrations/connectors/destination-redshift/src/test-integration/java/io/airbyte/integrations/destination/redshift/SshRedshiftDestinationBaseAcceptanceTest.java

```diff
@@ -0,0 +1,204 @@
+package io.airbyte.integrations.destination.redshift;
+
+import static io.airbyte.integrations.base.ssh.SshTunnel.TunnelMethod.SSH_KEY_AUTH;
+import static io.airbyte.integrations.base.ssh.SshTunnel.TunnelMethod.SSH_PASSWORD_AUTH;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableMap.Builder;
+import io.airbyte.commons.jackson.MoreMappers;
+import io.airbyte.integrations.standardtest.destination.comparator.TestDataComparator;
+import java.util.Map;
+import io.airbyte.commons.io.IOs;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.string.Strings;
+import io.airbyte.db.Database;
+import io.airbyte.db.factory.DSLContextFactory;
+import io.airbyte.db.factory.DatabaseDriver;
+import io.airbyte.db.jdbc.JdbcUtils;
+import io.airbyte.integrations.base.JavaBaseConstants;
+import io.airbyte.integrations.base.ssh.SshTunnel;
+import io.airbyte.integrations.destination.redshift.operations.RedshiftSqlOperations;
+import io.airbyte.integrations.standardtest.destination.JdbcDestinationAcceptanceTest;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public abstract class SshRedshiftDestinationBaseAcceptanceTest extends JdbcDestinationAcceptanceTest {
+
+  protected String schemaName;
+  // config from which to create / delete schemas.
+  protected JsonNode baseConfig;
+  // config which refers to the schema that the test is being run in.
+  protected JsonNode config;
+
+  private final RedshiftSQLNameTransformer namingResolver = new RedshiftSQLNameTransformer();
+  private final String USER_WITHOUT_CREDS = Strings.addRandomSuffix("test_user", "_", 5);
+
+  public abstract SshTunnel.TunnelMethod getTunnelMethod();
+
+  @Override
+  protected String getImageName() {
+    return "airbyte/destination-redshift:dev";
+  }
+
+  @Override
+  protected JsonNode getConfig() throws Exception {
+    final Map<Object, Object> configAsMap = deserializeToObjectMap(config);
+    final Builder<Object, Object> configMapBuilder = new Builder<>().putAll(configAsMap);
+    return getTunnelConfig(getTunnelMethod(), configMapBuilder);
+  }
+
+  protected JsonNode getTunnelConfig(final SshTunnel.TunnelMethod tunnelMethod, final ImmutableMap.Builder<Object, Object> builderWithSchema) {
+    final Path keyPath = Path.of("secrets/redshift-bastion-key-pair.pem");
```

Comment:
I think the key will have to be embedded in `config.json` in order to work with CI

Reply:
Fixed.